playground/nested.json

```diff
@@ -1,8 +1,995 @@
 {
-    "landing": {
-        "welcome": "Hello, welcome!"
-    },
-    "profile": {
-        "intro": "What is up for snacks later?"
+  "admin": {
```

Comment:
is this your project's json file? 😂

I would prefer it if this playground `nested.json` simpler and not as extensive. Let's keep it to max of 60 lines so that it's easier for users to understand it.